index.html

```diff
@@ -68,7 +69,204 @@ <h2> I’m a software developer</h2>
       <div class="chevron-btn">
         <img class="all-image" src="images/Action.svg" alt="arrow down image">
       </div>
+
+      <div class="text">
+        <h4>My Recent<br> Works</h4>
+      </div>
+
+      <div class="an-image">
+        <img src="images/Indicator.svg" alt="">
+      </div>
+    </section>
+
+    <section class="Portfolio">
+      <div class="portfolio-item">
+        <img class="portfolio-img" src="images/Laptop.jpg" alt />
+        <div class="portfolio-details">
+          <h2>Multi-Post Stories</h2>
+          <ul class="portfolio-ul">
+            <li class="portfolio-li ">Ruby on rails</li>
+            <li class="portfolio-li">css</li>
+            <li class="portfolio-li">Javascript</li>
+            <li class="portfolio-li">HTML</li>
+          </ul>
+          <button class="portfolio-btn" type="button">See Project</button>
+        </div>
+      </div>
+
+      <div class="portfolio-item">
+        <img class="portfolio-img" src="images/Laptop.jpg" alt>
+        <div class="portfolio-details">
+          <h2>Multi-Post Stories</h2>
+          <ul class="portfolio-ul">
+            <li class="portfolio-li">Ruby on rails</li>
+            <li class="portfolio-li">css</li>
+            <li class="portfolio-li">Javascript</li>
+            <li class="portfolio-li">HTML</li>
+          </ul>
+          <button class="portfolio-btn" type="button">See Project</button>
+        </div>
+      </div>
+
+      <div class="portfolio-item">
+        <img class="portfolio-img" src="images/Laptop.jpg" alt>
+        <div class="portfolio-details">
+          <h2>Multi-Post Stories</h2>
+          <ul class="portfolio-ul">
+            <li class="portfolio-li">Ruby on rails</li>
+            <li class="portfolio-li">css</li>
+            <li class="portfolio-li">Javascript</li>
+            <li class="portfolio-li">HTML</li>
+          </ul>
+          <button class="portfolio-btn" type="button">See Project</button>
+        </div>
+      </div>
+
+      <div class="portfolio-item">
+        <img class="portfolio-img" src="images/Laptop.jpg" alt>
+        <div class="portfolio-details">
+          <h2>Multi-Post Stories</h2>
+          <ul class="portfolio-ul">
+            <li class="portfolio-li">Ruby on rails</li>
+            <li class="portfolio-li">css</li>
+            <li class="portfolio-li">Javascript</li>
+            <li class="portfolio-li">HTML</li>
+          </ul>
+          <button class="portfolio-btn" type="button">See Project</button>
```

Comment:
- [ ] Kindly let us fix the color of these buttons (See project), for now, they don't look like what we have in the Figma design, see the difference below 👇 
- Your design
![image](https://user-images.githubusercontent.com/55465133/130302754-47a35543-8b76-4cdb-a621-0d552c0ad215.png)

- Figma design
![image](https://user-images.githubusercontent.com/55465133/130302780-929ee7c7-76a0-4910-94ae-1308fc7f8072.png)

- [ ] They should also follow the button styling: Active, Hover, and Pressed (see design) 👇 
![image](https://user-images.githubusercontent.com/55465133/130302828-d2b22c3a-4f2d-4c12-ac42-d8fa23c22e80.png)

Please let us fix these issues 😉